pkg/integration/tests/submodule/reset_folder.go

```diff
@@ -0,0 +1,110 @@
+package submodule
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var ResetFolder = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Reset submodule changes located in a nested folder.",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig:  func(cfg *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.EmptyCommit("first commit")
+		shell.CreateDir("dir")
+		shell.CloneIntoSubmodule("submodule1", "dir/submodule1")
+		shell.CloneIntoSubmodule("submodule2", "dir/submodule2")
+		shell.GitAddAll()
+		shell.Commit("add submodules")
+
+		shell.CreateFile("dir/submodule1/file", "")
+		shell.CreateFile("dir/submodule2/file", "")
+		shell.CreateFile("dir/file", "")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Files().Focus().
+			Lines(
+				Contains("▼ dir").IsSelected(),
```

Comment:
Please change all these to use Equals, following the convention we established in 0b5504aa98c5.